### PR TITLE
collapsable header

### PR DESCRIPTION
**Pull request by petrKavulok**
- 'user bar' (right part of navbar on screen > 800px) collapses when screen size drops below 800px. 
- sidebar sizing adjustments
- NavbarBrand position adjustments when sidebar is minimized

### CODE REVIEW

**Comment by petrKavulok** [comment]
https://user-images.githubusercontent.com/79644454/144863929-a9fbbef6-d3d5-4758-8d05-266f34cc4a0c.mov

File: src/containers/Header/index.js [hunk 1]
```diff
@@ -13,8 +13,11 @@ import NavbarBrand from './NavbarBrand';
 export function Header(props) {
 	const HeaderService = props.app.locateService("HeaderService");
 
+	const [userbarOpen, setUserbarOpen] = useState(false);
+
 	return (
-		<header className="application-header">
+		<header className={userbarOpen ? 'op' : ''}>
```
Comment (Pe5h4) [inline src/containers/Header/index.js:19]: @petrKavulok I think here you can simplify the condition this way

```
userBarOpen && "op"
```
Also, I would use better naming - 

`userbarOpen` >>> `headerProperties`
`op` >>> `open`

or something with similar meaning

File: src/containers/Header/index.js [hunk 2]
```diff
@@ -29,7 +32,7 @@ export function Header(props) {
 
 			{(props.app.props.hasSidebar || typeof props.app.props.hasSidebar === 'undefined') ? 
 				(
-					<Nav className="ml-auto" navbar>
+					<Nav className="ml-auto user-bar" navbar>
```
Comment (Pe5h4) [inline src/containers/Header/index.js:35]: @petrKavulok Also request for renaming

`user-bar` >>> `header-props`

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Anyway, it looks good! I would use better naming of variables and I am not sure about that `caret` toggler for displaying the header properties. Can you add some images how it would look like e.g. with some other icons like 
https://icons.coreui.io/icons/linear/chevron-circle-left-alt/
https://icons.coreui.io/icons/linear/chevron-circle-down-alt/

https://icons.coreui.io/icons/linear/chevron-left/
https://icons.coreui.io/icons/linear/chevron-bottom/

**Comment by petrKavulok** [comment]
I like the first one the most, the ones pointing to the side don't make much sense to me
https://user-images.githubusercontent.com/79644454/144999881-de261b8c-996c-4bd6-a346-07e51bb88245.mov


https://user-images.githubusercontent.com/79644454/144999889-c3f276bd-a75a-404d-81c8-df39e688025a.mov


https://user-images.githubusercontent.com/79644454/144999890-af6a0d1e-a409-497f-9f11-fc5bac6ea65a.mov


https://user-images.githubusercontent.com/79644454/145000845-91646c9e-5bed-4da4-b9f1-6d0bac45a597.mov

**Comment by ateska** [comment]
Number 1 looks cool to me.

**Review by ateska — APPROVED** [review]

**Review by Pe5h4 — APPROVED** [review]
Ok, looks fine. From what I have seen the first toggler was the best

**Pull request merged**